docker/docker-cluster/metrictank.ini

```diff
@@ -169,8 +187,18 @@ consumer-max-processing-time = 1s
 # How many outstanding requests a connection is allowed to have before sending on it blocks
 net-max-open-requests = 100
 
-## clustering transports ##
+## basic clustering settings ##
+[cluster]
+# The primary node writes data to cassandra. There should only be 1 primary node per shardGroup.
```

Comment:
Some comments end with a `.` and some don't. I'm fine either way, but maybe consistency would make a better impression.